editoast/src/core/path_properties.rs

```diff
@@ -79,6 +79,7 @@ pub struct OperationalPointOnPath {
     /// Distance from the beginning of the path in mm
     position: u64,
     /// Importance of the operational point
+    /// The value can be between 0 (less important) and 100 (more important)
     #[schema(required)]
```

Comment:
```suggestion
    /// Ascending importance of the operational point
    #[schema(required, minimum = 0, maximum = 100)]
```

OpenAPI already has fields for numeric bounds: https://swagger.io/docs/specification/v3_0/data-models/data-types/#numbers